subsystem/hcp/rosa_ocm_policies_data_source_test.go

```diff
@@ -0,0 +1,234 @@
+limitations under the License.
+*/
+package hcp
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2/dsl/core"             // nolint
+	. "github.com/onsi/gomega"                         // nolint
+	. "github.com/onsi/gomega/ghttp"                   // nolint
+	. "github.com/openshift-online/ocm-sdk-go/testing" // nolint
+)
+
+const (
+	// This is the cluster that will be returned by the server when asked to retrieve a cluster with ID 123
+	getStsPoliciesRequests = `{
+		"items": [
+			{
+				"kind": "STSPolicy",
+				"id": "openshift_cloud_credential_operator_cloud_credential_operator_iam_ro_creds_policy",
+				"details": "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSACloudCredentialOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "openshift_cloud_network_config_controller_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSACloudNetworkConfigOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "openshift_cluster_csi_drivers_ebs_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAClusterCSIDriversEBSOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "openshift_image_registry_installer_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAImageRegistryOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "openshift_ingress_operator_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSACloudIngressOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "openshift_machine_api_aws_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAMachineAPIOperator",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "shared_vpc_openshift_ingress_operator_cloud_credentials_policy",
+				"details":  "{}",
+				"arn": "",
+				"type": "OperatorRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "sts_installer_permission_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAInstallerPolicy",
+				"type": "AccountRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "sts_instance_controlplane_permission_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAControlPlanePolicy",
+				"type": "AccountRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "sts_instance_worker_permission_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSAWorkerPolicy",
+				"type": "AccountRole"
+			  },
+			  {
+				"kind": "STSPolicy",
+				"id": "sts_support_permission_policy",
+				"details":  "{}",
+				"arn": "arn:aws:iam::765374464689:policy/ROSASRESupportPolicy",
```

Comment:
I don't think we should add tests for those, they are for when we have managed policies on classic. Which will eventually fail these tests when we start updating the service to support it as the arn will change

Reply:
Oh this is mocked, then I'd rather not have the value with that account id on it

Reply:
@gdbranco 
I replaced the account id with mock numbers